README.md

```diff
@@ -173,6 +173,34 @@ Using the `Parameters` classes is not required, but they can help in a number of
 - Provides a more natural way to construct parameters
 - Facilitates ASP.NET Core model binding (bind an HTML form to a `Parameters` instance)
 
+## Debugging
+
+### View HTTP Requests
```

Comment:
suggestion: we should call this `HTTP Hooks` as the header and describe that they can view the requests in the details below. Hooks is a more universal language to use here and not everyone may want to view it, they may just want to be notified when requests/responses occur and do something totally different.

Additionally, this should probably go under the `usage` section instead as a sub section.

Reply:
Resolved